src/errorpage.cc

```diff
@@ -891,7 +891,7 @@ ErrorState::compileLogformatCode(Build &build)
 
         static MemBuf result;
         result.reset();
-        const auto logformatLen = Format::AssembleOne(logformat, result, ale);
+        const auto logformatLen = Format::AssembleOne(logformat, result, ale, RecordTime());
         assert(logformatLen > 0);
         const auto closure = logformat + logformatLen;
         if (*closure != '}')
```

Comment:
Please also fix these two legacy errorpage codes: `%t` and `%T`. No rush.

I think it would be acceptable to use RecordTime::legacySystemTime.tv_sec instead of time_t squid_curtime (which this PR does not save) for them:

```c++
    case 't':
        mb.appendf("%s", Time::FormatHttpd(squid_curtime));
        break;

    case 'T':
        mb.appendf("%s", Time::FormatRfc1123(squid_curtime));
        break;
```

Reply:
Done (e9693ce). I also fixed one more case here.